docs/theming/theme_scratch.md

```diff
@@ -174,24 +174,34 @@ http://localhost:8080/
 :alt: Add Plone site.
 ```
 
-Click on {guilabel}`Advanced` to show more options.
+Click on `Create Classic Plone site` to add your site.
 
-Scroll down to your package and activate the checkbox next to it. This will create a new Plone site and install our add-on in one go. Since the theme is part of the add-on it will be activated automatically.
+You see some basic styling because a precompiled `src/plonetheme/tokyo/theme/css/theme.min.css` has been shipped with the template.
 
-```{image} _static/scratch/add-plone-site-advanced.png
-:alt: Add Plone site advanced.
+Now we have to activate our add-on. This is done in the add-on section of the control panel:
+
+http://localhost:8080/Plone/@@overview-controlpanel
+
+```{image} _static/scratch/install-addon.png
+:alt: Install add-on.
 ```
 
-Click on `Create Plone site` to add your site.
+Navigate to the control panel and activate your add-on. After that step your site may look broken until we actually work on the new theme. Before we start theming we're going to add a copy of the main template to our theme package.
 
-You see some basic styling because a precompiled `src/plonetheme/tokyo/theme/css/theme.min.css` has been shipped with the template. Bevor we start theming we're going to add a copy of the main template to our theme package.
+```{note}
+You can always switch back to the default Barceloneta theme in the theming control panel: http://localhost:8080/Plone/@@theming-controlpanel
+```
 
 ## Override Main Template
 
 Copy the page template from `parts/omelette/Products/CMFPlone/browser/templates/main_template.pt` to `src/plonetheme/tokyo/browser/templates/main_template.pt`.
```

Comment:
I would use z3c.jbot to override the template. This way you do not need to change the zcml.

Reply:
Also I would mention they should make commit before modifying the template to have a clean diff.

Reply:
it does not work with jbot for the main_template as far as i know

Reply:
True, that's because of https://github.com/plone/Products.CMFPlone/issues/2666

Reply:
> Also I would mention they should make commit before modifying the template to have a clean diff.

I added it in the hints section.